cluster/terraform_kubernetes/ingress_controller.tf

```diff
@@ -87,5 +91,15 @@ resource "helm_release" "ingress-nginx-clone" {
       type  = set.value["type"]
     }
   }
+}
+resource "azurerm_public_ip" "public-ip" {
+  name                = "ingres-controller-pip"
```

Comment:
The name should be unique, use the resource prefix and the cluster environment name